Ah, we have a PlayStation EWS! But it's broken due to an unrelated problem:

```
CMake Error at C:/tools/cmake/share/cmake-3.31/Modules/Platform/PlayStation5.cmake:43 (message):
  Could not find Prospero SDK at C:/PS/SDK
Call Stack (most recent call first):
  C:/tools/cmake/share/cmake-3.31/Modules/CMakeDetermineSystem.cmake:146 (include)
  CMakeLists.txt:10 (project)
  ```

--- a/Source/WebKit/Platform/IPC/unix/ConnectionUnix.cpp
+++ b/Source/WebKit/Platform/IPC/unix/ConnectionUnix.cpp
@@ -130,9 +130,8 @@ void Connection::platformInvalidate()
 #if USE(GLIB)
     m_socket = nullptr;
 #else
-    if (m_socketDescriptor != -1)
-        closeWithRetry(m_socketDescriptor);
-    m_socketDescriptor = -1;
+    if (m_socketDescriptor.value() != -1)
+        closeWithRetry(m_socketDescriptor.release());
 #endif
 
     if (!m_isConnected)

